### PR TITLE
Two More Changes

### PR DESCRIPTION
**Pull request by perkinsjr**
I added a call out and code block to the first section.

I also cleaned up the markdown plugin section

### CODE REVIEW

File: content/guides/tina-cloud/getting-started/markdown-plugin.md [hunk 1]
```diff
@@ -27,29 +27,30 @@ yarn add react-tinacms-editor
...
-            }}
+            clientId={process.env.NEXT_PUBLIC_TINA_CLIENT_ID}
+            branch={process.env.NEXT_PUBLIC_EDIT_BRACH}
+            organization={process.env.NEXT_PUBLIC_ORGANIZATION_NAME}
+            isLocalClient={Boolean(
+              Number(process.env.NEXT_PUBLIC_USE_LOCAL_CLIENT ?? true)
+            )}
++            cmsCallback={cms => {
++              +              cms.plugins.add(MarkdownFieldPlugin)
+                          }}
```
Comment (logan-anderson) [inline content/guides/tina-cloud/getting-started/markdown-plugin.md:50]: Should there be a + on this line as well

File: content/guides/tina-cloud/getting-started/markdown-plugin.md [hunk 2]
```diff
@@ -27,29 +27,30 @@ yarn add react-tinacms-editor
...
-              cms.plugins.add(MarkdownFieldPlugin)
-            }}
+            clientId={process.env.NEXT_PUBLIC_TINA_CLIENT_ID}
+            branch={process.env.NEXT_PUBLIC_EDIT_BRACH}
+            organization={process.env.NEXT_PUBLIC_ORGANIZATION_NAME}
+            isLocalClient={Boolean(
+              Number(process.env.NEXT_PUBLIC_USE_LOCAL_CLIENT ?? true)
+            )}
++            cmsCallback={cms => {
++              +              cms.plugins.add(MarkdownFieldPlugin)
```
Comment (logan-anderson) [inline content/guides/tina-cloud/getting-started/markdown-plugin.md:49]: It also might be "best practice" to lazy load this witch looks like this.
```
cmsCallback={cms => {
              import('react-tinacms-editor').then((field)=>{
                cms.plugins.add(field.MarkdownFieldPlugin)
              })
            }}
```

The way you have it now is fine too just might increase the users bundle size (a decen amount).

Comment (perkinsjr) [reply content/guides/tina-cloud/getting-started/markdown-plugin.md:49]: Jeff redid this section happy to LL though let me change it !

File: content/guides/tina-cloud/getting-started/markdown-plugin.md (hunk 1 above)
Comment (perkinsjr) [reply content/guides/tina-cloud/getting-started/markdown-plugin.md:50]: 👎

File: content/guides/tina-cloud/getting-started/create-app.md [hunk 3]
```diff
@@ -19,8 +19,17 @@ We created a quick way to bootstrap a Tina application to show the power of visu
...
 
-> Heads up, this will override the `_app.js`, which has a stylesheet in it. Add it back to the top of the file -- `import "../styles/index.css";`
+> Heads up, this will override the `_app.js`, which has a stylesheet in it. Make sure to add it back otherwise the styles will be broken.
 
+Your import section should now look like
+
+```diff 
+import dynamic from 'next/dynamic'
+import { TinaEditProvider } from 'tinacms/dist/edit-state'
++ import "../styles/index.css"
```
Comment (DirtyF) [inline content/guides/tina-cloud/getting-started/create-app.md:29]:
```suggestion
+ import '../styles/index.css'
```

Comment (DirtyF) [reply content/guides/tina-cloud/getting-started/create-app.md]: Original Next.js starter and other imports use single quote

File: content/guides/tina-cloud/getting-started/markdown-plugin.md [hunk 4]
```diff
@@ -27,29 +27,32 @@ yarn add react-tinacms-editor
 
 Inside the `pages/_app.js` file we need to import the `MarkdownFieldPlugin` and add it to the CMS callback:
 
-```jsx
-import '../styles/index.css'
+```diff
 import dynamic from 'next/dynamic'
 import { TinaEditProvider } from 'tinacms/dist/edit-state'
+import "../styles/index.css"
```
Comment (DirtyF) [inline content/guides/tina-cloud/getting-started/markdown-plugin.md:33]:
```suggestion
import '../styles/index.css'
```

**Review by logan-anderson — APPROVED** [review]

**Pull request merged**